### DISCUSSION TITLE
Submenu border color? Help!

### DISCUSSION THREAD

**Discussion by Hybrid-Divide** [Open-Shell/Open-Shell-Menu#1980] [Q&A] [answered]
I've made some progress, and now have my menu width the way I want it, along with the submenu offset!

But I noticed that the submenu border color is brighter than the main menu, and I'd like to make them both the same shade of grey as the main menu, or be able to get rid of them altogether.  But I've looked through the Start Manu Skinning guide, and see nothing regarding borders.

Please help!
See the screenshot to see the difference in border colors between the grey of the main menu, and the white of the submenu.
Thanks.
![image](https://github.com/user-attachments/assets/2be2062a-9d19-4daf-80ae-b86b0a3c5d1a)

**Comment by ge0rdi** [comment, marked as answer]
You should be able to adjust color of the frame using `Submenu_bitmap_tint1` element.

**Reply by Hybrid-Divide** [reply]
That worked great. Thank you!
